.github/workflows/publish_release_artifacts_npm.yml

```diff
@@ -0,0 +1,53 @@
+name: "Publish :: NPM :: Release"
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  build_and_publish:
+    if: github.repository == 'kiegroup/kogito-editors-java'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # This bash script returns the `tag` name for the release. Will match "/refs/[tags/heads]/[tag]"
+      - name: Parse tag
+        id: release_tag
+        run: |
+          echo ::set-output name=tag::$(node -e "console.log('${{ github.ref }}'.match(/^.*\/(.+)$/)[1])")
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Build with Maven
+        run: mvn clean install -B -ntp --file pom.xml
+
+      - name: Publish BPMN Editor
+        run: |
+          cd ${{ github.workspace }}/.github/supporting-files/publish-artifacts-npm/bpmn-editor-unpacked
+          cp -r ${{ github.workspace }}/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-kogito-runtime/target/kie-wb-common-stunner-bpmn-kogito-runtime ./target/bpmn
+          ls -la .
+          npm version ${{ steps.release_tag.outputs.tag }}
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.KIEGROUP_NPM_TOKEN }}" > ~/.npmrc
+          npm publish --access public
+
+      - name: Publish DMN Editor
+        run: |
+          cd ${{ github.workspace }}/.github/supporting-files/publish-artifacts-npm/dmn-editor-unpacked
+          cp -r ${{ github.workspace }}/kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/target/kie-wb-common-dmn-webapp-kogito-runtime ./target/dmn
+          ls -la .
+          npm version ${{ steps.release_tag.outputs.tag }}
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.KIEGROUP_NPM_TOKEN }}" > ~/.npmrc
+          npm publish --access public
+
+      - name: Publish SceSim Editor
+        run: |
+          cd ${{ github.workspace }}/.github/supporting-files/publish-artifacts-npm/scesim-editor-unpacked
+          cp -r ${{ github.workspace }}/drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/target/drools-wb-scenario-simulation-editor-kogito-runtime ./target/scesim
+          ls -la .
```

Comment:
Leftover?

Reply:
Removed in https://github.com/kiegroup/kogito-editors-java/pull/10/commits/e1d415c9b20289832eca6a5380fc0be2d9ae0377.